Code/Source/RobotImporter/URDF/JointsMaker.cpp

```diff
@@ -28,21 +29,27 @@ namespace ROS2
         }
     }
 
-    void JointsMaker::AddJoint(urdf::JointSharedPtr joint, AZ::EntityId childEntityId, AZ::EntityId parentEntityId)
+    bool JointsMaker::HasRequiredComponentsForJoint(AZ::EntityId entityId)
     {
-        AZ::Entity* childEntity = AzToolsFramework::GetEntityById(childEntityId);
-        AZ::Entity* parentEntity = AzToolsFramework::GetEntityById(parentEntityId);
+        AZ::Entity* entity = AzToolsFramework::GetEntityById(entityId);
+        return entity->FindComponent<PhysX::EditorColliderComponent>()
+            ? true
+            : entity->FindComponent<PhysX::EditorShapeColliderComponent>() != nullptr;
```

Comment:
Increase readability?

```suggestion
return (entity->FindComponent<PhysX::EditorColliderComponent>() != nullptr || entity->FindComponent<PhysX::EditorShapeColliderComponent>() != nullptr);
```